osu.Game/Rulesets/BindableRulesetSelector.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Allocation;
+using osu.Framework.Bindables;
+
+namespace osu.Game.Rulesets
+{
+    public abstract class BindableRulesetSelector : RulesetSelector
```

Comment:
As I mentioned before, I'm not sure why the bindable isn't inside `RulesetSelector` itself. There are two situations I foresee:

1. Something like toolbar that references the global ruleset:

```
class Toolbar:
    void load(OsuGameBase game):
        Children = new[]
        {
            ...
            /* some derived class that adds styling */
            new ToolbarRulesetSelector { Current = { BindTarget = game.Ruleset } }
        }
```

2. Something like osu!direct that has a local ruleset:

```
class DirectOverlay:
    void load():
        Children = new[]
        {
            ...
            /* some derived class that adds styling */
            rulesetSelector = new DirectRulesetSelector()
        }

        rulesetSelector.Current.BindValueChanged(_ => { /* change local ruleset */ }, true);
```